code/game/objects/machinery/computer/nuke_disk_generator.dm

```diff
@@ -200,3 +200,20 @@
 	icon_state = "nuke_blue"
 	disk_type = /obj/item/disk/nuclear/blue
 	disk_color = "blue"
+
+//list of disk gens we want to spawn for nuke related gamemodes. Could just use subtypes, but using a list in case more gen related stuff is added in the future
+GLOBAL_LIST_INIT(nuke_disk_generator_types, list(/obj/machinery/computer/nuke_disk_generator/red, /obj/machinery/computer/nuke_disk_generator/green, /obj/machinery/computer/nuke_disk_generator/blue))
+
+/obj/structure/computer3frame/nuke_disk_candidate
```

Comment:
why is this a computer3frame child

Reply:
Copy pasted for stats/icon, forgot to change it. Updated now.